README.md

```diff
@@ -28,6 +28,9 @@ Below is a curated list of awesome digital tools for activism. These are tools f
  * [TakeCharge.io](https://github.com/controlshift/prague-server) - opensource or SaaS online donations powered by Stripe.
  * [VictoryKit](http://www.victorykitapp.com) - Optimize petition campaign content. Not actively maintained.
  * [Countdown](https://github.com/drewrwilson/countdown) - A quick & simple tool for making your own webpage with a countdown to your upcoming event.
+ * [DemocracyOS](http://democracyos.org) - An open-source platform for public policy engagement, open feedback, and participatory governance. 
```

Comment:
I just tried it out and looks like there's a heroku error on this site :(

Is this a project you're working on? If so, let me know when you get it back up again and I'll review it again. 

In the mean time I'll look at your other two suggestions

![image](https://user-images.githubusercontent.com/4324656/55023075-6653b880-4fd2-11e9-93e4-28f6031a96b1.png)